tests/storage/cdi_clone/conftest.py

```diff
@@ -12,13 +12,15 @@ def cirros_dv_with_filesystem_volume_mode(
     unprivileged_client,
     namespace,
     storage_class_with_filesystem_volume_mode,
+    storage_profile_minimum_supported_pvc_size,
 ):
     yield from create_cirros_dv(
         client=unprivileged_client,
         namespace=namespace.name,
         name="cirros-fs",
         storage_class=storage_class_with_filesystem_volume_mode,
         volume_mode=DataVolume.VolumeMode.FILE,
+        dv_size=storage_profile_minimum_supported_pvc_size,
```

Comment:
When we create a DV with '`storage`' api - `minimumSupportedPvcSize` should be applied automatically, no?